### PR TITLE
OWC2020 News and Wiki article

### PR DESCRIPTION
**Pull request by WalterToro**
_oh no._

### CODE REVIEW

File: wiki/Tournaments/OWC/2020/en.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
+---
+tags:
+  - OWC 2020
```
Comment (TPGPL) [inline wiki/Tournaments/OWC/2020/en.md:3]:
```suggestion
  - OWC
```

File: wiki/Tournaments/OWC/2020/en.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | US$300 per team member, exclusive single-run merch, profile badge, "osu! Champion" user title for one year |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | US$160 per team member, exclusive single-run merch, profile badge |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | US$80 per team member, exclusive single-run merch, profile badge |
+
+## Organisation
+
+The osu! World Cup 2020 is run by the [osu! team](/wiki/People/The_Team) and various community members.
```
Comment (TPGPL) [inline wiki/Tournaments/OWC/2020/en.md:37]:
```suggestion
The osu! World Cup 2020 is run by the osu!team and various community members.
```
i dont think there's a need to add the same wikilink like above

File: wiki/Tournaments/OWC/2020/en.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+| Referee | ![][flag_ES] [Deif](https://osu.ppy.sh/users/318565), ![][flag_AU] [ill onion](https://osu.ppy.sh/users/8306102), ![][flag_BR] [LeoFLT](https://osu.ppy.sh/users/3668779), ![][flag_NL] [nik](https://osu.ppy.sh/users/10077264), ![][flag_DE] [p3n](https://osu.ppy.sh/users/123703), ![][flag_US] [tigereyes144](https://osu.ppy.sh/users/6499811), ![][flag_CL] [WalterToro](https://osu.ppy.sh/users/5281416), ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973) |
+| Statistician | ![][flag_BR] [LeoFLT](https://osu.ppy.sh/users/3668779), ![][flag_DE] [Nwolf](https://osu.ppy.sh/users/1910766) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1150567)
+- [Livestream](https://www.twitch.tv/osulive)
+- **[Click here to sign-up!](https://osu.ppy.sh/community/tournaments/26)**
```
Comment (TPGPL) [inline wiki/Tournaments/OWC/2020/en.md:51]:
```suggestion
- **[Registration form](https://osu.ppy.sh/community/tournaments/26)**
```
might be better?

File: wiki/Tournaments/OWC/2020/en.md [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+[flag_AU]: /wiki/shared/flag/AU.gif
+[flag_BR]: /wiki/shared/flag/BR.gif
+[flag_CA]: /wiki/shared/flag/CA.gif
+[flag_CL]: /wiki/shared/flag/CL.gif
+[flag_DE]: /wiki/shared/flag/DE.gif
+[flag_ES]: /wiki/shared/flag/ES.gif
+[flag_GB]: /wiki/shared/flag/GB.gif
+[flag_KR]: /wiki/shared/flag/KR.gif
+[flag_NL]: /wiki/shared/flag/NL.gif
+[flag_US]: /wiki/shared/flag/US.gif
```
Comment (TPGPL) [inline wiki/Tournaments/OWC/2020/en.md:182]:
```suggestion
[flag_AR]: /wiki/shared/flag/AR.gif "Argentina"
[flag_AT]: /wiki/shared/flag/AT.gif "Austria"
[flag_AU]: /wiki/shared/flag/AU.gif "Australia"
[flag_BR]: /wiki/shared/flag/BR.gif "Brazil"
[flag_CA]: /wiki/shared/flag/CA.gif "Canada"
[flag_CL]: /wiki/shared/flag/CL.gif "Chile"
[flag_DE]: /wiki/shared/flag/DE.gif "Germany"
[flag_ES]: /wiki/shared/flag/ES.gif "Spain"
[flag_GB]: /wiki/shared/flag/GB.gif "United Kingdom"
[flag_KR]: /wiki/shared/flag/KR.gif "South Korea"
[flag_NL]: /wiki/shared/flag/NL.gif "Netherlands"
[flag_US]: /wiki/shared/flag/US.gif "United States"
```

File: news/2020-09-24-OWC-2020-registrations-open.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+---
+
+We bashed the drums, catched the fruits, hit the notes, but now is, once again, time to ***click the circles!***. The OWC 2020 is here and registrations are now open!
+
+![](/wiki/shared/news/banners/OWC_2020_banner.jpg)
+
+We are keeping the structure introduced last year with some changes. A Qualifier Stage will define the top 32 teams who will play on the Main Bracket Stage, so if you think you have the skills to compete, is now the time to register.
+
+For details, refer to the **[osu! World Cup 2020 wiki article](/wiki/Tournaments/OWC/2020)**.
+You can discuss this event and follow the most important changes in the **[official discussion thread](https://osu.ppy.sh/community/forums/topics/1150567)**.
```
Comment (TPGPL) [inline news/2020-09-24-OWC-2020-registrations-open.md:14]:
```suggestion
For details, refer to the **[osu! World Cup 2020 wiki article](/wiki/Tournaments/OWC/2020)**. You can discuss this event and follow the most important changes in the **[official discussion thread](https://osu.ppy.sh/community/forums/topics/1150567)**.
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**